### ISSUE TITLE
Missing addon files in r158 tarball

### ISSUE THREAD

**Issue by Michsior14** [three-types/three-ts-types#657]
-   `three` version: 0.158.0
-   `node` version: 20.8.1
-   `npm` (or `yarn`) version: yarn 4.0.1

### Problem description:

For some reason the latest published tarball is missing some addons files (even though they were not removed in the code base). This can be observed in `node_modules` or directly on npm: https://www.npmjs.com/package/@types/three?activeTab=code.

### Relevant code:

The list missing paths highlighted by my IDE in `Addons.d.ts` file:
```ts
export * from './controls/FirstPersonControls.js';
export * from './geometries/TextGeometry.js';
export { default as IESSpotLight } from './lights/IESSpotLight.js';
export * from './loaders/FontLoader.js';
export * from './loaders/HDRCubeTextureLoader.js';
export * from './loaders/IESLoader.js';
export * from './loaders/RGBMLoader.js';
export * from './modifiers/TessellateModifier.js';
export * from './objects/BatchedMesh.js';
export * from './objects/GroundProjectedSkybox.js';
export * from './objects/MarchingCubes.js';
export * from './shaders/ExposureShader.js';
export * from './shaders/MMDToonShader.js';
export * from './shaders/ToonShader.js';
export * from './textures/FlakesTexture.js';
export * as CameraUtils from './utils/CameraUtils.js';
export * from './utils/ShadowMapViewer.js';
export * from './utils/TextureUtils.js';
export * from './utils/WorkerPool.js';
```

### Suggested solution:

Not sure, seems it's an publish issue?

**Comment by Methuselah96** [comment]
Thanks for the issue! This seems to do with the fact that these files are no longer referenced by tests (as of the latest release) and also not referenced in OTHER_FILES, with means DT doesn't publish them. The CI used to fail in these scenarios, but I'm guessing that changed in the recent DT updates. I'll reach out to them and get this sorted out.

**Comment by Methuselah96** [comment]
@Michsior14 This should be fixed in @types/three@0.158.1 by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67249, can you verify that it's working for you now?

**Comment by Michsior14** [comment]
@Methuselah96 Looks good now, thx :)
